tools/reboot.sh

```diff
@@ -23,6 +23,7 @@ do_bootstrap(){
 
         flux bootstrap "$provider" \
             --owner="$owner" \
+            --components-extra=image-reflector-controller,image-automation-controller \
```

Comment:
Does this mean that image automation will need to be installed for Weave GitOps to work going forward?

Reply:
That'd be very nice. Progressive enhancement ftw.

Reply:
Good point, if it's causing UI errors perhaps we can put this behind a flag that the user can enable.

Reply:
Yes we should handle this gracefully and show a message that explains why the image automation list is empty (when the CRD is not installed). Strictly speaking this is not an error because the controllers are not expected to be installed (in the majority of cases).